exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -190,10 +193,13 @@ public QProfiles(List<ProfileInfo> runningQueries, List<ProfileInfo> finishedQue
     public List<String> getErrors() { return errors; }
   }
 
+  //max Param to cap listing of profiles 
+  private static final String MAX_QPROFILES_PARAM = "max";
```

Comment:
This is to define the param in the URL. This would be too verbose to type into the URL IMO.